blocks/product-assortment-carousel-block/features/product-assortment-carousel/default.jsx

```diff
@@ -145,7 +151,12 @@ function ProductAssortmentCarousel({ customFields = {} }) {
 										href={item.attributes ? getProductURL(item?.attributes) : "#"}
 										assistiveHidden={viewable ? null : true}
 									>
-										<Image alt={item.name ? "" : item.name} src={item.image} />
+										<Image
```

Comment:
As we are updating the image - should we add some sizes to this to avoid rendering the original image?

Reply:
yeah can do. will put some out there. any suggestions?

Reply:
That's a known issue with resizer until they add the new option to support aspect ratio resizing

Reply:
ah gotcha

Reply:
right -- I vaguely remember Nick talking about this as well

Reply:
Updated based off of product feedback as well